ale/base/data_isis.py

```diff
@@ -297,6 +268,14 @@ def __enter__(self):
         """
         return self
 
+    def __exit__(self, exc_type, exc_val, exc_tb):
```

Comment:
Not sure if this is better than just erroring out. If you're using an ISIS based driver you don't need either the __enter__ or __exit__

Reply:
This allows for usage both inside and outside of a context. Adding this just makes the usage syntax the same across of the drivers.